tests/core/test_accounts.py

```diff
@@ -94,12 +100,17 @@ def PRIVATE_KEY_ALT(request):
 
 
 @pytest.fixture
-def acct(request):
+def acct():
```

Comment:
Is there much point to having this fixture?  Seems to just return a reference to a class that's already in scope.

Reply:
Hm, yeah, I think it used to supply both the instance and the class... I guess that got dropped at some point. Investigating...

Reply:
Looks like it still works as a class or instance, when I added that back in, so I did.